ichnaea/db.py

```diff
@@ -156,6 +157,12 @@ def __init__(self, uri, pool=True, transport="default"):
             options["connect_args"]["ssl_version"] = PROTOCOL_TLSv1
             # Needed for SSL
             options["connect_args"]["ssl_ca"] = certifi.where()
+        elif DB_TRANSPORTS[transport] == 'pymysql':
+            # PyMySQL 8.0 changed for compatibility with mysqlclient
+            # Use 7.0's binary prefixes, for selecting and inserting binary data
```

Comment:
I think you mean `0.8.0` and `0.7.0` here.

Reply:
yep 🤦‍♂